src/loaders/ObjectLoader.js

```diff
@@ -311,8 +311,10 @@ THREE.ObjectLoader.prototype = {
 			for ( var i = 0, l = json.length; i < l; i ++ ) {
 
 				var image = json[ i ];
+				var isAbsolute = SCHEME.test( image.url );
```

Comment:
`isAbsolute` is true for absolute URLs and false for relative ones. However, I've updated the PR and put the regex and isAbsolute check inline. Feel free to request more changes! :-)